.travis.yml

```diff
@@ -32,8 +32,10 @@ install:
   - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
   - if [ "$DEPENDENCIES" == "low" ]; then composer update --prefer-lowest; fi;
 
+before_script:
+  - echo "<?php if (PHP_VERSION_ID >= 50400) echo ',@php5.4';" > php_version_tags.php
```

Comment:
There was an issue on HHVM when inlining it at the time @everzet did this for Behat